README.md

```diff
@@ -104,11 +105,43 @@ Hakyll is going to build full URLs for them. Happy coding!
 Here is an example of the pull request that introduces the
 `hakyll-shortcut-links` library into the scope:
 
-* [Some Hakyll website](https://github.com/vrom911/vrom911.github.io/pull/32)
+- [Some Hakyll website](https://github.com/vrom911/vrom911.github.io/pull/32)
 
 As you can see by the diff the only actual change needed for that was replacing
 `pandocCompiler` with `allShortcutLinksCompiler`.
 
+#### Examples of the `@github` shortcut :
+
+- Link to a user :
+  ```
+  [kowainik](@github:kowainik)
+  [kowainik](https://github.com/kowainik)
+  ```
+  ```
+  [kowainik](@github(kowainik))
+  [kowainik](https://github.com/kowainik)
+  ```
+  ```
+  [kowainik](@github)
+  [kowainik](https://github.com/kowainik)
+  ```
+- Link to a repository :
+
+  ```
+  [hakyll-shortcut-links](@github:kowainik)
+  [hakyll-shortcut-links](https://github.com/kowainik/hakyll-shortcut-links)
+  ```
+
+  ```
+  [hakyll-shortcut-links](@github(kowainik))
+  [hakyll-shortcut-links](https://github.com/kowainik/hakyll-shortcut-links)
+  ```
+
+  ```
+  [kowainik/hakyll-shortcut-links\](@github)
+  [hakyll-shortcut-links](https://github.com/kowainik/hakyll-shortcut-links)
+  ```
```

Comment:
I don't think we need 6 examples since there only 4 possible combinations. By using different strings, it's possible to show the complete behaviour of all options using 4 examples in total.

Reply:
so which examples should i remove ?

Reply:
I see that the following shortcuts are used twice: `@github`, `@github(kowainik)`, `@github:kowainik`.
At the same time, there's no example on the `@github(foo):bar` usage.

Reply:
Okay 👌 i'll use a  update with`foo bar`words